packages/flutter/test/widgets/selectable_region_test.dart

```diff
@@ -3594,6 +3594,20 @@ void main() {
     skip: kIsWeb, // [intended] Web uses its native context menu.
   );
 
+  testWidgets('Ensure consistent layout of SelectionArea children between platforms', (WidgetTester tester) async {
+    await tester.pumpWidget(
+      const MaterialApp(
+        home: SelectionArea(
+          child: Text('Row 1'),
+        ),
+      ),
+    );
+    await tester.pumpAndSettle();
+    final Offset offset = tester.getTopLeft(find.byType(Text));
+    expect(offset, Offset.zero);
+  },
```

Comment:
nit: I think its okay to place these last two lines together on one line without a comma like below

```dart
});
```